### PR TITLE
Adds detection for Qwant Mobile iOS browser

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

**Comment by sanchezzzhak** [comment]
@sgiehl Is it acceptable to split the browser into two names? 
Similar precedent: `Opera Mini iOS`, `Firefox Mobile iOS`

**Comment by sgiehl** [comment]
@sanchezzzhak  sorry for the delay. Good question. I guess I actually would say no. We should detect that as one, to keep it simple. There is actually no benefit in splitting it. We maybe should even consider removing the other iOS browser versions. Or do you have another opinion on that?

**Comment by sanchezzzhak** [comment]
I am in favor of not specifying prefixes

File: regexes/client/browsers.yml [hunk 1]
```diff
@@ -635,6 +635,11 @@
     default: 'Blink'
 
 # Qwant Mobile
+- regex: 'QwantiOS/(\d+[\.\d]+)'
+  name: 'Qwant Mobile iOS'
```
Comment (sanchezzzhak) [inline regexes/client/browsers.yml:639]: change name: `Qwant Mobile`

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**